.github/workflows/nightly-test.yaml

```diff
@@ -59,8 +68,19 @@ jobs:
         uses: actions/upload-artifact@v4
         with:
           name: ${{ env.artifact_name }}
-          path: ${{ github.workspace }}/inspection-reports.tar.gz
+          path: ${{ github.workspace }}/inspection-reports_${{ env.os_dash }}_${{ env.release_dash }}_${{ matrix.arch }}.tar.gz
       - name: Tmate debugging session
         if: ${{ failure() && github.event_name == 'pull_request' }}
         uses: mxschmitt/action-tmate@v3
         timeout-minutes: 10
+      - name: Extract CNCF conformance report
+        if: ${{ failure() && matrix.release == 'latest/edge/moonray' }}
+        working-directory: tests/integration
+        run: |
+         tar -xf sonobuoy_e2e.tar.gz --one-top-level
+      - name: Upload CNCF conformance report artifact
+        uses: actions/upload-artifact@v4
+        if: ${{ failure() && matrix.release == 'latest/edge/moonray' }}
```

Comment:
We are only uploading failed runs from moonray. Is that the intended behavior or is this for debugging only? 
I would assume that we would like to see the conformance report uploaded in all cases where we ran the conformance test.

Reply:
but we run conformance test only on moonray because it is only one that passes cncf tests. it will change in the future